<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>415</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>415</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>415</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	background-color: rgb(8,189,253);
}

.QWidget
{
	
	background-color: rgb(255, 255, 255);
}

QPushButton
{
	border:none;
}

QPushButton#btnMin
{
	image: url(:/images/24gl-minimization.png);
}

QPushButton#btnClose
{
	image: url(:/images/close.png);
}

QPushButton#btnMin:hover
{
	background-color: rgb(165, 165, 165);
}

QPushButton#btnClose:hover
{
	background-color: rgb(255, 0, 0);
}

QPushButton#btnLogin
{
	background-color: rgb(8,189,253);
	border-radius:5px;
	font: normal 22px &quot;楷体&quot;;
	color: white;
}
QPushButton#ptn_cancel
{
	background-color: rgb(8,189,253);
	border-radius:5px;
	font: normal 22px &quot;楷体&quot;;
	color: white;
}

QPushButton#btnLogin:hover
{
	background-color: rgba(8,189,253,220);
}

QPushButton#btnQrcode
{
	image: url(:/images/Qrcode.png);
}

QPushButton#btnQrcode:hover
{
	image: url(:/images/blackQrcode.png);
}

QComboBox, QLineEdit
{
	border:none;
	border-bottom:1px solid rgb(299, 299, 299);
	font: normal 18px &quot;楷体&quot;;
	color: rgb(193, 193, 193);
}

QComboBox:hover, QLineEdit:hover
{
	border-bottom:1px solid rgb(193, 193, 193);
}

QComboBox:focus, QLineEdit:focus
{
	border-bottom:1px solid rgb(18, 183, 245);
}

QComboBox::drop-down
{
	min-width:25px;
	border-image: url(:/images/down_arrow.png);
}

QComboBox::drop-down:hover
{
	border-image: url(:/images/adown_arrow.png);
}

QComboBox::drop-down:checked
{
	border-image: url(:/images/up_arrow.png);
}
</string>
  </property>
  <widget class="QLabel" name="labtop">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>540</width>
     <height>160</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>540</width>
     <height>160</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/number.jpg</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>540</width>
     <height>261</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>415</height>
    </size>
   </property>
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>300</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>45</height>
     </size>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="username">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>300</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>35</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>35</height>
     </size>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>2020984130603</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <item>
     <property name="text">
      <string>2020984130603</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1360701</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>300</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>35</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>35</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>321</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="Auto_Log">
       <property name="text">
        <string>自动登录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="Rem_Passwd">
       <property name="text">
        <string>记住密码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRegister">
       <property name="text">
        <string>找回密码</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="ptn_cancel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>140</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <zorder>ptn_cancel</zorder>
   <zorder>btnLogin</zorder>
   <zorder>username</zorder>
   <zorder>password</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
  <widget class="QPushButton" name="btnMin">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>0</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>0</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
